src/app/productConfig/productConfigForm/productConfigForm.component.js

```diff
@@ -292,6 +292,12 @@ class ProductConfigFormController {
     this.updateQueryParam({ key: giveGiftParams.month, value: month })
   }
 
+  changeComments () {
+    if (this.itemConfig['donation-services-comments'] != '') {
+      this.itemConfig['recipient-comments'] = ''
```

Comment:
I am a bit concerned about essentially deleting everything the donor has typed already if they start adding donation-services-comments.  But it's important that we remove recipient-comments from itemConfig if there are donation-serivces-comments.  Maybe I can temporarily store recipient-comments locally, so that they can be re-loaded if donation-services-comments are deleted?

Reply:
That's an interesting thought. It would certainly be a more complicated solution. Maybe we get this live, then go back and do that? How much effort do you think it would be to have that logic?

Reply:
It might not be so bad.... it would probably be handled inside this method here, at some point I would store the value of `itemConfig['recipient-comments']` inside of `this`, and I'll have another conditional here that will bring that value back to `itemConfig['recipient-comments']` once `itemConfig['donation-services-comments']` is empty.  It could get complex making sure the logic is sound, though.

Reply:
You can take a quick pass at it if you want. Even if there are edge cases, at least some people would get their comments back. You might want them in session storage vs being in memory, so if they go back after navigating around, the comments will still be there.